pkg/asset/installconfig/powervs/session.go

```diff
@@ -49,25 +40,27 @@ type UserCredentials struct {
      4) put it into Platform {userid: , iamtoken: , ...}
 */
 func GetSession() (*Session, error) {
-	s, uc, err := getPISession()
+	s, err := getPISession()
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to load credentials")
 	}
 
-	return &Session{Session: s, Creds: uc}, nil
+	return &Session{Session: s}, nil
 }
 
 /*
 //  https://github.com/IBM-Cloud/power-go-client/blob/master/ibmpisession/ibmpowersession.go
 */
-func getPISession() (*ibmpisession.IBMPISession, *UserCredentials, error) {
+func getPISession() (*ibmpisession.IBMPISession, error) {
 
 	var (
 		id, passwd, apikey, region, zone string
```

Comment:
i don't think you need apikey *and* passwd. you can just remove passwd

Reply:
Changing passwd to iamtoken and setting iamtoken to apikey for clarity